cmake/ctest_script.cmake

```diff
@@ -426,9 +514,9 @@ macro(run_ctests)
   endif()
   #set(CTEST_LABELS_FOR_SUBPROJECTS ${LABELS_FOR_SUBPROJECTS}) #!Dangerous!
 
-  ctest_configure(OPTIONS "${CMAKE_ARGS};-DCTEST_USE_LAUNCHERS=${CTEST_USE_LAUNCHERS};-DBUILD_TESTING=ON;-DDART_TESTING_TIMEOUT=${CTEST_TEST_TIMEOUT}")
+  ctest_configure(OPTIONS "${CMAKE_ARGS};${SUBM_ARGS};-DCTEST_USE_LAUNCHERS=${CTEST_USE_LAUNCHERS};-DBUILD_TESTING=ON;-DDART_TESTING_TIMEOUT=${CTEST_TEST_TIMEOUT}")
 
-  ctest_submit(PARTS Configure Update)
+  ctest_submit(PARTS Configure Update Notes)
```

Comment:
This happens immediately, or?

Reply:
I did not think of this, will make a new commit to make sure that it is the case.